modules/querying/pages/query-operations.adoc

```diff
@@ -1142,3 +1142,99 @@ The scope of the command depends on the user's current scope.
+
+=== Update description
+After creating a query, you can add a description for the query.
+
+*Syntax*
+[source]
+----
+UPDATE DESCRIPTION OF <objectType> <objectName> [ON GRAPH <graphName>] <description>
+----
+
+Required privilege
+`WRITE_QUERY`
+
+To update a description, run the `UPDATE DESCRIPTION` command as shown below. You can update the description of a query or a query parameter. If you choose not to specify a graph in the command, the current scope will be used as the scope of the description.
+
+[source, gsql]
+----
+GSQL > USE GRAPH poc_graph
+GSQL > UPDATE DESCRIPTION OF QUERY query1 "List of all employees in the company"
+GSQL > UPDATE DESCRIPTION OF QUERY query2 ON GRAPH graph1 "Details of employees in a specific department"
+GSQL > UPDATE DESCRIPTION OF QUERY_PARAM query1.param1 "Department where the employee works"
+----
+
+[NOTE]
+====
+The term `UPDATE` here means adding a description to the target.
+If the target does not already have a description, it will be added.
+If the target already has a description, it will be replaced with the new one.
+====
+
+[CAUTION]
+====
+* When updating a query, use `CREATE OR REPLACE QUERY` instead of `CREATE QUERY` to retain the query description.
+* `CREATE QUERY` may unintentionally drop both the query and its description if it already exists, whereas `CREATE OR REPLACE QUERY` ensures the description is retained by replacing the query without dropping it.
+====
+
+=== Show description
+
+*Syntax*
+[source]
+----
+SHOW DESCRIPTION OF <objectType> <objectName(s)> [ON GRAPH <graphName>]
+----
+
+`<objectName(s)>` can include a comma-separated list of names.
+
+Required privilege
+`READ_QUERY`
+
+To display a description, use the `SHOW DESCRIPTION` command as shown below. Similar to the update command, you can display the description of a query or a query parameter.
+
+
+[source, gsql]
+----
+GSQL > USE GRAPH poc_graph
+GSQL > SHOW DESCRIPTION OF QUERY query1
+GSQL > SHOW DESCRIPTION OF QUERY query1,query2
+GSQL > SHOW DESCRIPTION OF QUERY * ON GRAPH graph1
+GSQL > SHOW DESCRIPTION OF QUERY_PARAM query1.param1
+GSQL > SHOW DESCRIPTION OF QUERY_PARAM query1.param1,query1.param2
+GSQL > SHOW DESCRIPTION OF QUERY_PARAM query2.*
+----
+
+Support is provided for displaying descriptions of multiple targets simultaneously. You can separate the targets using commas, or use the wildcard `*` to display all descriptions.
+
+=== Drop description
+
+*Syntax*
+[source]
+----
+DROP DESCRIPTION OF <objectType> <objectName> [ON GRAPH <graphName>]
+----
+
+Required privilege
+`WRITE_QUERY`
+
+To remove a description, run the `DROP DESCRIPTION` command as shown below. This command can be used to remove the description of both global and local type objects, as well as queries and query parameters.
+
+[source, gsql]
+----
+GSQL > USE GRAPH poc_graph
+GSQL > DROP DESCRIPTION OF QUERY query1
+GSQL > DROP DESCRIPTION OF QUERY query1*,query2*
```

Comment:
`GSQL > DROP DESCRIPTION OF QUERY query1*,query2*`

I think we have another question for a developer. 
What does `query1*` mean?  It seems to say "a name that begins with `query1` and then has any other characters. For example, `query1A`, `query1_is_the_best` and `query1_is_my_favorite` would fit this pattern.

In all the other examples, `*` could only be used to represent an entire name, not part of a name. My guess is that either `*` actually cannot be used for part of a name OR it ALWAYS can be used for the suffix of a name. We need the developer to clarify